topics/full-stack-development-with-kotlin-multiplatform.topic

```diff
@@ -321,19 +293,48 @@
         </p>
         <p>
             These clients are not linked to the Ktor service at the moment, but they do use the shared
-            <code>Greeting</code> class. For example, if you execute the
-            <ui-path>iosApp</ui-path>
-            run configuration, a client will start
-            in the iOS Simulator:
-            <img style="block" src="full_stack_development_tutorial_run_android.gif"
-                 alt="Running the app in the Android Simulator" width="300" border-effect="rounded"/>
-            This shows a screen with a single button placed on the top. Pressing the button shows and hides an image.
-            When the button is pressed for the first time, the details of the current platform are added to its text.
-            The code to achieve this is found in
-            <path>composeApp/src/commonMain/kotlin/App.kt</path>
-            :
+            <code>Greeting</code> class.
         </p>
-        <code-block lang="kotlin"><![CDATA[
+    </chapter>
+    <chapter title="Run the client application" id="run-client-app">
+        <p>
+            You can run the client application by executing the
+            <a href="https://www.jetbrains.com/help/idea/run-debug-configuration.html#launch-run-configuration">
+                run configuration
+            </a>
+            for the target. To run the application on an iOS simulator, follow the steps below:
+        </p>
+        <procedure id="run-ios-app-procedure">
+            <step>
+                In IntelliJ IDEA, select the
+                <path>iosApp</path>
+                run configuration and a simulated device.
+                <img src="full_stack_development_tutorial_run_configurations.png"
+                     alt="Run &amp; Debug window" width="400"
+                     border-effect="line" style="block"/>
+            </step>
+            <step>
+                Click on the
+                <ui-path>Run</ui-path>
+                button
+                (<img src="intellij_idea_run_icon.svg"
+                      style="inline" height="16" width="16"
+                      alt="IntelliJ IDEA run icon"/>)
+                to run the configuration.
+            </step>
+            <step>
+                <p>
+                    When you run the iOS app, it is built with Xcode under the hood and launched in the iOS Simulator.
+                    This shows a screen with a single button placed on the top. Pressing the button shows and hides an
+                    image.
+                    <img style="block" src="full_stack_development_tutorial_run_ios.gif"
+                         alt="Running the app in the iOS Simulator" width="300" border-effect="rounded"/>
+                    When the button is pressed for the first time, the details of the current platform are added
```

Comment:
On staging, the spacing between the GIF and the following text is small compared to other spacings, maybe separate this text in its own <p>?